doc/content/source/materials/ConcreteThermalMoisture.md

```diff
@@ -376,73 +396,101 @@ mechanisms separately. Instead, they tend to reproduce the general combined tren
+
+  where $\alpha_h$, $\beta_h$ and $\gamma_h$ are coefficients from test data. Since the value of the humidity diffusivity coefficient for aggregates, $D_{Hagg}$, typically
+  is negligible compared with the value of $D_{Hcp}$, it is assumed to be zero in the current implementation. Note that this model is applicable for concrete with w/c > 0.5.
+
+  Since no coupling between heat and moisture transfer is considered the value of $D_{ht}$, $C_{a}$, are $C_{w}$ considered as zero.
 
 2. [!cite](bazant1982finite)
 
-!equation id=Dh0_bazant
-D_{h,0} = D_1f_H
+  [!cite](bazant1982finite) describe the moisture diffusivity as a function of temperature and relative humidity according to
+
+!equation id=Dh_bazant
+D_h = \left\{\begin{array}{lll}
+      D_1 f_{1H} f_{2T} & \text{for} & T \le 95 \degree\text{C} \\
+      D_1 f_{2T}(T=95 \degree C) f_{3T} & \text{for} & T > 95 \degree\text{C}
+      \end{array}\right.
 
-where $D_1 = 3.10 \times 10^{-10}m^2/s$ and
+where $D_1 = 3.10 \times 10^{-10}m^2/s$ and f_{1H}, f_{2T}, and f_{3T} are given by
 
 !equation id=fH
-f_H = \left\{\begin{array}{lll}
-        \alpha_D + \frac{1-\alpha_D}{1+\left(\frac{1-H}{1-0.75}\right)^n} & \text{for} & T \le 95\degree C\\
-        1 & \text{for} & T > 95 \degree C
+f_{1H} = \left\{\begin{array}{lll}
+        \alpha_D + \frac{1-\alpha_D}{1+\left(\frac{1-H}{1-0.H_c}\right)^n} & \text{for} & H \le 1\\
+        1 & \text{for} & H > 1
     \end{array}\right.
 
-Also, $f_H = 1$ when $H > 1$ and $\alpha_D$ is given by
+!equation id=f2T
+f_2(T) = \exp\left(\frac{Q}{R}\left(\frac{1}{(T_{ref} + 273.15)}-\frac{1}{(T + 273.15)}\right)\right)
+
+!equation id=f3T
+\begin{array}{lll}
+    f_3(T) = \exp\left(\frac{T-95}{0.881 + 0.214(T-95)}\right) & \text{for} & T > 95 \degree C
+\end{array}
+
+where $\alpha_D$ is given by
 
-!equation id=D_bazant
+!equation id=alfaD_bazant
 \alpha_D = \frac{1}{1+\frac{19(95-T)}{70}}
 
-where $T$ is in $\degree$C. $\alpha \in [0.037:1]$ from $T\in[0\degree C : 95\degree C]$ and $n\in[6:16]$.
+$Q$ is activation energy for water migration along the adsorption layers in the necks, and $R$ is gas constant with $Q/R$=2700 K, and $T$ is in $\degree$C. $\alpha \in [0.037:1]$ from $T\in[0\degree C : 95\degree C]$ and $n\in[6:16]$.
 
-It's obvious that all three reference moisture diffusivity models strongly depend on the value of humidity $H$, and indirectly on the temperature $T$. Once the value of
-reference moisture diffusivity $D_{h,0}$ is obtained, the actual concrete moisture diffusivity $D_h$ required by the moisture diffusion governing equation, [!eqref](moisture_governing),
-can then be calculated by
+It has been reported by [!cite](bazant1982finite) that the additional moisture diffusion due to thermal gradients included in the moisture governing equation is negligible. Thus the value of $D_{ht}$ is set to $1.0\times10^{-5} \times D_h$ by default. This parameter can, however, be set to an arbitrary value if desired.
```

Comment:
I assume that 1e-5 is a small value, but probably worth saying that explicitly.

Reply:
Yes, it is small. I could not find any reference for this value.